main.js

```diff
@@ -41,12 +51,17 @@ client.on('message', message => {
 
     if (!client.commands.has(command)) return;
 
+    if(command === 'status'){
```

Comment:
#8 No command is needed for this. Just set the activity status in `client.once()`